.github/workflows/build_windows_artifacts.yml

```diff
@@ -145,8 +151,18 @@ jobs:
           ccache -z
           python3 build_tools/github_actions/build_configure.py
 
-      - name: Build therock-archives and therock-dist
-        run: cmake --build "${{ env.BUILD_DIR }}" --target therock-archives therock-dist -- -k 0
+      - name: Build therock-archives and therock-dist with memory monitoring
```

Comment:
Step name doesn't really need "with memory monitoring", especially if memory monitoring is not always enabled